src/Pages/FieldInfoPage.js

```diff
@@ -295,6 +322,17 @@ function FieldInfoPage() {
           />
         }
       ></SimpleModal>
+      {/* modal to delete selected field data */}
+      <SimpleModal
```

Comment:
Can we rename the component to DeleteConfirmationDialog or something more context specific?

Reply:
it's common to edit and delete, and can be reused wherever we need a center aligned pop-up